code/modules/surgery/organs/vocal_cords.dm

```diff
@@ -288,7 +288,7 @@
 		cooldown = COOLDOWN_STUN
 		for(var/mob/living/carbon/C in listeners)
 			if(C.silent < VOICE_OF_GOD_MAX_SILENCE_TIME)
-				C.silent = min(C.silent + (10 SECONDS * power_multiplier), VOICE_OF_GOD_MAX_SILENCE_TIME)
+				C.silent = min(C.silent + (10 * power_multiplier), VOICE_OF_GOD_MAX_SILENCE_TIME)
```

Comment:
Doesn't removing the SECONDS define reduce the added silent time by an order of magnitude?

Reply:
`SECONDS` is a macro that multiplies the number by 10, aka converts seconds into deciseconds.

The `silent` variable, however, is measured in _seconds_, not deciseconds, unlike most of everything else.